lib/tests-tree-builder/base.ts

```diff
@@ -184,23 +201,6 @@ export class BaseTestsTreeBuilder {
         }
     }
 
-    protected _addNodeId(parentSuiteId: string, nodeId: string, {fieldName}: {fieldName: 'browserIds' | 'suiteIds'}): void {
```

Comment:
This method is called "addNodeId", so it have to check for duplicates with slow "_isNodeIdExists"
But in "_addSuites" we usually don't need to do it because we process suites from parent to child, and quick "tree.suites.byId[childSuiteId]" is enough